mappings/net/minecraft/world/gen/VeinCreator.mapping

```diff
@@ -0,0 +1,21 @@
+CLASS net/minecraft/class_6955 net/minecraft/world/gen/VeinCreator
```

Comment:
I suggest `class_6955` → `OreVeinSampler` and `method_40548` → `create`. This way, `class_6955.method_40548` can be read as a method that returns a sampler (and that sampler samples a block state in a ore vein for a given position).